docs/docs/product/roadmap.mdx

```diff
@@ -579,23 +579,21 @@ Excitingly, v3 introduces the foundational elements for Actions V2, opening up a
     <details>
         <summary>Actions V1</summary>
     </details>
-</details>
-
-<details>
-    <summary>Breaking Changes</summary>
-
-    <BreakingChanges/>
 
     <details>
-        <summary>Hosted Login v1 will be removed</summary>
+        <summary>Hosted Login v1 will be deprecated</summary>
     </details>
 
 
     <details>
-        <summary>Zitadel APIs v1 will be removed</summary>
+        <summary>Zitadel APIs v1 will be deprecated</summary>
     </details>
 </details>
```

Comment:
Should we make a bullet list instead of collapsable items without content?

<img width="1378" height="609" alt="image" src="https://github.com/user-attachments/assets/1dde7777-6b85-4f70-96af-36dae298b798" />